README.md

```diff
@@ -77,6 +77,8 @@ Create a multi-line plot to show the total service request counts by `created_da
 
 At Data Engineering, we enhance datasets with geospatial attributes, such as point locations and administrative boundaries. To help us better understand the data from **Python Task 1**, we would like you to join the initial raw data to an NYC administrive boundary. Then create a choropleth map of the 7 day total count of complaints where `HPD` is the responding agency fot a specific `complaint_type` of your choice.
```

Comment:
"we would like you to join the initial raw data to an NYC administrive boundary other than borough"

Reply:
"(such as Neighborhood Tabulation Area (NTA), Community District (CD))" - not sure if that is necessary but I think overall the older wording is more clear

Reply:
I checked the open data site and community district is assigned as well. I don't want to list an administrative boundaries they should assign to as I think we should test proficiency with these terms and how to find a good geospatial dataset to join to

Reply:
Added a commit to list geospatial boundaries they can't use

Reply:
Cool - I like this